spec/services/user_deletion_service_spec.rb

```diff
@@ -79,14 +79,37 @@
           expect(responder.reload.deleted_at).not_to be nil
         end
 
-        it 'unassigns the cases from user' do
+        it 'unassigns the live cases from user without touching responded cases' do
           kase = create :accepted_case,
                         responder: responder,
                         responding_team: team
+
+          responded_case = create :responded_case,
+                                  responder: responder,
+                                  responding_team: team,
+                                  received_date: 5.days.ago
+          expect(kase.responder_assignment.state).to eq "accepted"
```

Comment:
We could wrap these expectation blocks in methods to make it a bit more readable:

`responder_has_accepted_initial_draft_case`
`responder_has_accepted_responded_case`

etc...